draft-iab-protocol-maintenance.md

```diff
@@ -299,10 +332,9 @@ protocol errors:
   deployments are tolerant of the same non-compliant behavior.
 
 This happens because interoperability requirements for protocol implementations
-are set by other deployments. Specifications and - where they exist -
-conformance test suites might guide the initial development of implementations,
-but implementations ultimately need to interoperate with deployed
-implementations.
+are set by other deployments. The IETF does not create nor mandate conformance
+test suites for protocols, causing established implementations to become
+de-facto conformance test suites used by new implementations.
```

Comment:
This was a comment from Stuart that I had written a while back in #41. I've removed it from this PR and we can discuss it there since it's orthogonal.